robusta_krr/core/models/objects.py

```diff
@@ -13,12 +13,22 @@ def __hash__(self) -> int:
         return hash(self.name)
 
 
+class HPAData(pd.BaseModel):
+    min_replicas: int
```

Comment:
I think this field might be optional
https://www.manifests.io/kubernetes/1.26/io.k8s.api.autoscaling.v1.HorizontalPodAutoscalerSpec?linked=HorizontalPodAutoscaler.spec